Tests/Editor/SettingsAnalysisTests.cs

```diff
@@ -237,5 +237,45 @@ public void SettingsAnalysis_GraphicsUsingRenderingPath_WithBuiltinRenderPipelin
             GraphicsSettings.defaultRenderPipeline = defaultRenderPipeline;
 #endif
         }
+
+        [Test]
+        public void SettingsAnalysis_FogEnable_IsReported()
+        {
+            var fog = RenderSettings.fog;
+            RenderSettings.fog = true;
+
+            var issues = Analyze(IssueCategory.ProjectSetting, i => i.descriptor.id.Equals("PAS1003"));
+            var playerSettingIssue = issues.FirstOrDefault();
+
+            Assert.NotNull(playerSettingIssue);
+
+            RenderSettings.fog =fog;
```

Comment:
Very minor style:  I would add a space after the "=" assignment symbol.

Reply:
(line removed)